Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -358,6 +359,7 @@ final class MiscellaneousTestCase: XCTestCase {
     }
 
     func testUnicode() async throws {
+        try XCTSkipOnWindows()
```

Comment:
chore (blocking): Can we please add a "because" indicating why the test is being skipped?